source/fundamentals/bson.txt

```diff
@@ -226,6 +226,8 @@ This example performs the following actions:
     to use the ``"json"`` struct tag if a ``"bson"`` struct tag is not specified
   - Sets the ``NilSliceAsEmpty`` field to ``true``, which instructs the driver
     to marshal ``nil`` Go slices as empty BSON arrays
+  - Sets the ``OmitEmpty`` field to ``true``, which instructs the driver to globally omit
```

Comment:
S: can you also add a note on the work with bson page about the ability to set this option? can link back to this section

Reply:
Which page are you referring to? This is the Work with BSON page: https://deploy-preview-477--docs-golang.netlify.app/fundamentals/bson/

Reply:
Oh I didnt realize ~ Then add a link to the struct tag section

Reply:
Added a link in the struct tag section's ``omitempty`` information! Let me know what you think.